specs/signature-specification.md

```diff
@@ -230,6 +230,83 @@ The Notary Project facilitates signing arbitrary blobs using detached signatures
 1. Apart from the `payload` field, all other fields of a signature envelope are identical between OCI and blob signatures.
 1. OCI signatures have signature manifest files, containing the mediaType of the signature envelope (application/jose+json or application/cose). However, blob signatures lack such manifest file and this mediaType information is instead found in the signature file extension.
 
+### Distribute arbitrary blob and blob signature with an OCI conformant registry
+Besides local file system, an arbitrary blob and its blob signature can also be stored in an OCI conformant registry.
+
+Notary Project client supports discover and verify such blob signatures with following requirements:
+1. The target arbitrary blob MUST be stored in the OCI registry as an OCI image. This [OCI image's manifest](https://github.com/opencontainers/image-spec/blob/v1.1.0/manifest.md) MUST have `"mediaType": "application/vnd.oci.image.manifest.v1+json"`. 
+2. This OCI image manifest MUST have one and only one layer, which describes the target arbitrary blob. 
```

Comment:
Looks like the blob file in the OCI registry is just a normal OCI image manifest, right?